lib/Jsrt/Jsrt.cpp

```diff
@@ -4625,6 +4625,27 @@ CHAKRA_API JsTTDReplayExecution(_Inout_ JsTTDMoveMode* moveMode, _Out_ int64_t*
 #endif
 }
 
+CHAKRA_API JsTTDDiagSetAutoTraceStatus(_In_ bool status)
+{
+#if !ENABLE_TTD
+    return JsErrorCategoryUsage;
+#else
+    JsrtContext *currentContext = JsrtContext::GetCurrent();
+    JsErrorCode cCheck = CheckContext(currentContext, JSRT_MAYBE_TRUE);
+    TTDAssert(cCheck == JsNoError, "Must have valid context when setting auto trace status.");
```

Comment:
Is TTDAssert a release assert? If not, probably should return the error here?

Reply:
Looks like it's currently a fatal error in release.  It seems like we should just return the error rather than killing the process and let the calls abort if something's wrong.